cedar-policy/src/frontend/is_authorized.rs

```diff
@@ -656,9 +825,47 @@ impl RecvdSlice {
 #[cfg(test)]
 mod test {
     use super::*;
-    use crate::{frontend::utils::assert_is_failure, EntityUid};
     use cool_asserts::assert_matches;
-    use std::collections::HashMap;
+    use serde_json::json;
+
+    /// Assert that `is_authorized_json()` returns Allow with no errors
+    #[track_caller] // report the caller's location as the location of the panic, not the location in this function
```

Comment:
Is it considered a best practice to have this comment every time? It feels redundant to me.

Reply:
Can remove :)